osgp/protocol-adapter-mqtt/osgp-protocol-simulator-mqtt/src/main/java/org/opensmartgridplatform/simulator/protocol/mqtt/Default.java

```diff
@@ -0,0 +1,6 @@
+package org.opensmartgridplatform.simulator.protocol.mqtt;
+
+public class Default {
+    public static final String BROKER_HOST = "127.0.0.1";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this IP "127.0.0.1" address configurable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1313)

Reply:
Leaving as is since this is not deployed to production.